docs/rest_api.rst

```diff
@@ -1945,6 +1945,1154 @@ getasynctaskresponse.json
+|                    | }                                                                                                          |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Success Response   | Code: 200                                                                                                  |
+|                    |                                                                                                            |
+|                    | Content: <Success message>                                                                                 |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Example Response   | {                                                                                                          |
+|                    |                                                                                                            |
+|                    | "code": 200,                                                                                               |
+|                    |                                                                                                            |
+|                    | "message": bigip-fast-templates-service-discovery template set deleted successfully.                       |
+|                    |                                                                                                            |
+|                    | }                                                                                                          |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Error Response     | Code: 400                                                                                                  |
+|                    |                                                                                                            |
+|                    | Content: {error: Bad request}                                                                              |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Notes              |                                                                                                            |
++--------------------+------------------------------------------------------------------------------------------------------------+
+
+
+searchendpoint.json
+`````````````````````
+
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Title              | Searches Endpoint across login BIG-IPs                                                                     |
```

Comment:
(not exactly sure what this means, but maybe:)
Searches Endpoints across logged in BIG-IPs
(maybe login BIG-IPs means something else, if so, leave it)